### ISSUE TITLE
Carpet Sky Additions 1.20

### ISSUE THREAD

**Issue by Drongeorange** [jsorrell/CarpetSkyAdditions#122]
Will Carpet Sky Additions be updated to version 1.20

**Comment by ANDREJ6693** [comment]
^^^ pls

**Comment by intcreator** [comment]
when I try to run the 1.19.4 carpet sky editions with fabric 1.20.1 I get this error:
```
[18:25:44] [ServerMain/WARN]: Missing data pack fabric
[18:25:45] [ServerMain/ERROR]: Unsupported key: carpetskyadditions:skyblock; Unsupported key: carpetskyadditions:skyblock; Unsupported key: carpetskyadditions:skyblock
```

**Comment by jsorrell** [comment]
Soon

**Comment by Kluski42** [comment]
> Soon

Do you plan to create a 1.20 compatible version of the 1.19.4 skyblock first (adding the new features later), or do you intend to update to 1.20 alongside all the new features?

**Comment by intcreator** [comment]
personally I'd prefer a quick fix that allows the game to run in 1.20 first; I upgraded my Fabric server to 1.20 and it doesn't work but I'm worried if I try to go back to 1.19 it will break things even more. I'd rather get up and running in 1.20 with no new features even if it takes longer to add any new features later

**Comment by jsorrell** [comment]
> personally I'd prefer a quick fix that allows the game to run in 1.20 first

A 1.20 compatible version without new features has been uploaded. A version with the new features will be released soon.
